pkg/apis/batch/types.go

```diff
@@ -238,6 +238,20 @@ type CronJobList struct {
 	Items []CronJob
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CronJobManualInstantiation represents a manual triggering of a CronJob.
+type CronJobManualInstantiation struct {
```

Comment:
I was thinking this will be called `CronJobManualRequest` or `CronJobRequest`, for example, wdyt?
@kubernetes/api-reviewers thoughts?

Reply:
I spoke with @smarterclayton the suggestion is either `CronJobManualRequest` or `CronJobInstantiate`.